webcompat/static/css/development/components/hero.css

```diff
@@ -52,23 +52,25 @@
     display: inline-block;
     position: relative;
     font-size: 2em;
-    color: #414042;
     text-decoration: none;
-    transition: top 100ms ease-out 0ms;
-    top:1em;
+    top: 1em;
+    margin-top: 2.4em;
+    padding: 0.4em 0.6em;
```

Comment:
Sorry I'm not the review, but if you can remove `0`  before all `.`

`padding: .4em .6em`